ui/admin/app/components/form/credential/json/index.hbs

```diff
@@ -48,8 +48,8 @@
   </Hds::Form::Textarea::Field>
 
   {{#if @model.isNew}}
-    <form.fieldset>
-      <:body>
+    <Hds::Form::Fieldset @layout='horizontal' as |F|>
```

Comment:
Is this fieldset needed?

Reply:
@ZedLi good point, we are just displaying the radio card here.